packages/react-ds/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react'
+
+import classNames from 'classnames/bind'
+
+import { IInput } from '@venice/core/models/input.model'
+import styles from '@venice/styles/components/Input.module.scss'
+
+const Input: FC<IInput> = ({
+  id,
+  label,
+  name,
+  style,
+  className,
+  error,
+  ...props
+}: IInput) => {
+  const selfId = id || `input__${name}`
+
+  return (
+    <div
+      className={classNames(
+        styles.container,
+        { [styles.invalid]: !!error },
+        className
```

Comment:
Can you move the `classNames` method outside the return syntax and use a variable with the returned value here?